Makefile-package.mk

```diff
@@ -11,54 +14,36 @@ VENDOR             = "New Relic, Inc."
 PACKAGER           = "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
 PACKAGE_URL        = "https://www.newrelic.com/infrastructure"
 SUMMARY            = "New Relic Infrastructure Integrations"
-DESCRIPTION        = "New Relic Infrastructure Integrations extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from your infrastructure components such as MySQL,\nNGINX and Cassandra."
-FPM_COMMON_OPTIONS = --verbose -C $(SOURCE_DIR) -s dir -n $(PROJECT_NAME) -v $(VERSION) --iteration $(RELEASE) --prefix "" --license $(LICENSE) --vendor $(VENDOR) -m $(PACKAGER) --url $(PACKAGE_URL) --config-files /etc/newrelic-infra/ --description "$$(printf $(DESCRIPTION))" --depends "newrelic-infra >= 1.0.726" --depends "nrjmx"
-FPM_DEB_OPTIONS    = -t deb -p $(PACKAGES_DIR)/deb/ --replaces "newrelic-infra-integrations-beta (<= 1.0.0)"
-FPM_RPM_OPTIONS    = -t rpm -p $(PACKAGES_DIR)/rpm/ --epoch 0 --rpm-summary $(SUMMARY) --replaces "newrelic-infra-integrations-beta <= 1.0.0"
 
-package: create-bins prep-pkg-env $(PACKAGE_TYPES)
+define DESCRIPTION
+New Relic Infrastructure Integrations extend the core New Relic\n\
+Infrastructure agent\'s capabilities to allow you to collect metric and\n\
+live state data from your infrastructure components such as MySQL,\n\
+NGINX and Cassandra.\n\
+\n\
+This is a meta-package that specifies dependencies to the\n\
+aforementioned integrations, and installs all of them. You may prefer\n\
+to install them individually, selecting only those that you need.
+endef
+
+FPM_COMMON_OPTIONS = --verbose -C tmp -s dir -n $(PROJECT_NAME) -v $(VERSION) --iteration $(RELEASE)\
+ --prefix "" --license $(LICENSE) --vendor $(VENDOR) -m $(PACKAGER) --url $(PACKAGE_URL)\
+ --description "$$(printf $(DESCRIPTION))"\
+ --depends $(DEP_APACHE) --depends $(DEP_CASSANDRA) --depends $(DEP_MYSQL) --depends $(DEP_NGINX) --depends $(DEP_REDIS)
 
```

Comment:
I'd keep these two dependencies: 
--depends "newrelic-infra >= 1.0.726" --depends "nrjmx"

Reply:
Won't it consider the dependency anyway? (coming it transitively from the integrations dependencies)

Reply:
yes, I thought it after submitting the comment.